oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceSchemaReader.java

```diff
@@ -84,7 +87,11 @@ public Schema getSchema(ResultSetMetaData metadata, int index) throws SQLExcepti
         return Schema.of(Schema.LogicalType.TIMESTAMP_MICROS);
       case Types.TIMESTAMP:
```

Comment:
For timestamp it should be return Schema.of(Schema.LogicalType.DATETIME); always